image-caption-generator-web-app.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: image-caption-generator-web-app
+spec:
+  selector:
+    matchLabels:
+      app: image-caption-generator-web-app
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: image-caption-generator-web-app
+    spec:
+      containers:
+      - name: image-caption-generator-web-app
+        image: codait/image-caption-generator-web-app:latest
```

Comment:
The image name on Dockerhub is [`codait/max-image-caption-generator-web-app`](https://hub.docker.com/r/codait/max-image-caption-generator-web-app/)

Reply:
@lresende is there a reason you changed the naming conventions to include `max` when adding the web app repo compared to the model repos? I can always change the yaml file

Reply:
All the other images on https://hub.docker.com/u/codait/ are also `max-`, so that does seem to be the convention